srcpkgs/libgdal/template

```diff
@@ -1,27 +1,63 @@
 # Template file for 'libgdal'
 pkgname=libgdal
-version=3.0.4
-revision=9
+version=3.2.3
+revision=1
+# aarch & arm temporary disabled
+archs="~aarch* ~armv*"
 wrksrc="gdal-${version}"
 build_style=gnu-configure
-configure_args="--with-liblzma --with-webp --with-zstd --with-podofo --with-opencl=yes"
-hostmakedepends="gettext-devel pkg-config python-numpy json-c-devel"
-makedepends="freexl-devel geos-devel jasper-devel json-c-devel
- libcurl-devel libopenexr-devel libopenjpeg2-devel libpodofo-devel libqhull-devel
- libwebp-devel libxml2-devel libzstd-devel netcdf-devel opencl2-headers pcre2-devel
- proj-devel sqlite-devel ocl-icd-devel"
+configure_args="
+ --with-expat=yes
+ --with-liblzma=yes
+ --with-opencl=yes
+ --with-podofo=yes
+ --with-spatialite=yes
+ --with-sqlite3=yes
+ --with-webp=yes
+ --with-zstd=yes
+ $(vopt_if kml libkml)
+ $(vopt_with postgresql pg)"
+hostmakedepends="gettext-devel pkg-config python3-numpy json-c-devel swig"
+makedepends="boost-devel expat-devel freexl-devel geos-devel jasper-devel
+ json-c-devel libcurl-devel libopenexr-devel libopenjpeg2-devel
+ libpodofo-devel libqhull-devel libspatialite-devel libwebp-devel
+ libxml2-devel libzstd-devel netcdf-devel ocl-icd-devel opencl-headers
+ pcre2-devel proj-devel python3-devel sqlite-devel
+ $(vopt_if kml libkml-devel)
+ $(vopt_if postgresql postgresql-libs-devel)"
 short_desc="Geospatial Data Abstraction Library"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Nyx70 <n.y.x@bluewin.ch>"
 license="MIT"
 homepage="http://www.gdal.org/"
-distfiles="http://download.osgeo.org/gdal/${version}/gdal-${version}.tar.xz"
-checksum=5569a4daa1abcbba47a9d535172fc335194d9214fdb96cd0f139bb57329ae277
-subpackages="libgdal-devel libgdal-tools"
+distfiles="https://github.com/OSGeo/gdal/releases/download/v${version}/gdal-${version}.tar.gz"
+checksum=86a35aad60a1eb87c2c0c145f9bccd83a47c4781254544ed5246f64d55ee1f18
+subpackages="python3-gdal libgdal-devel libgdal-tools"
+
+build_options="kml postgresql"
+build_options_default="kml"
 
 if [ -z "$CROSS_BUILD" ]; then
 	makedepends+=" hdf5-devel"
 fi
 
+do_build() {
+	: ${make_cmd:=make}
+	${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}
+	# python modules
+	rm -f swig/python/*_wrap.cpp
+	${make_cmd} -C swig/python generate
+	cd swig/python
+	${make_cmd} ${makejobs} PYTHON=python3 ${makejobs} ${make_build_args} ${make_build_target}
+}
+
+do_install() {
+	: ${make_cmd:=make}
+	${make_cmd} DESTDIR=${DESTDIR} install
+	# python modules
+	cd swig/python
+	${make_cmd} PYTHON=python3 DESTDIR=${DESTDIR}/ install
```

Comment:
same as with `do_build` move the last 2 lines to `post_install` and do not overwrite `do_install` at all
```bash
	# python modules
	cd swig/python
	make PYTHON=python3 DESTDIR=${DESTDIR}/ install
```